src/main/target/ALIENFLIGHTF3/target.c

```diff
@@ -64,37 +63,28 @@ const uint16_t airPWM[] = {
 
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
-    //
     // 6 x 3 pin headers
-    //
+    { TIM15, IO_TAG(PB15), TIM_Channel_2, TIM1_BRK_TIM15_IRQn,     1, IOCFG_AF_PP, GPIO_AF_1 }, // PWM1  - PB15 - TIM1_CH3N, TIM15_CH1N, *TIM15_CH2
```

Comment:
I presume you mean in all targets.

Yes,  eventually we should change that, but it's a big edit, so I don't think it should hold up this merge.

Once this merge is in the `target.c` files will be **identical** to the files in betaflight - so we can have confidence that they have been tested.